### ISSUE TITLE
Android xml resource (strings.xml) containing tags aren't parsed correctly

### ISSUE THREAD

**Issue by spocky** [mozilla/pontoon#3448]
[As described here](https://developer.android.com/guide/topics/resources/localization?hl=en) Android resources can contain xliff tags. For example :
```
<string name="shortcuts_additional_settings">
  <xliff:g id="vendor" example="Xiaomi">%1$s</xliff:g> additional settings
</string>
```

Unfortunately, when they do, the synchronized resource is not parsed correctly. With the example above, the text (appearing in the left column) will contain the whole string tag, and not only its content.

<img width="948" alt="Capture d'écran 2024-11-19 004407" src="https://github.com/user-attachments/assets/65941ac4-63c7-4995-8e4c-71ba23713ebd">

A workaround for a correct parsing would be to include content in CDATA, but this would then lead to Android studio not replacing %1$s with example data, as it does with the initial syntax.

Is there any way to make the xml parser not fail and output the whole tag instead of only its content as it normally should ?

**Comment by mathjazz** [comment]
Thanks for the report! We're in the process of switching file parsers and serializers to our new [`moz.l10n`](https://github.com/mozilla/moz-l10n/) library, which will allow Pontoon to support cases like this.

**Comment by spocky** [comment]
Is there any PR I could follow to check the progress regarding this migration and be alerted when this will be supported ?

**Comment by mathjazz** [comment]
Please see #3538.

@eemeli Would you like this issue to remain open?

**Comment by eemeli** [comment]
Yeah, this should remain open as a bug we'll hopefully be able to fix.
